app/uk/gov/hmrc/gform/sharedmodel/formtemplate/Expr.scala

```diff
@@ -40,6 +40,17 @@ final case class FormCtx(formComponentId: FormComponentId) extends Expr
 final case class AddressLens(formComponentId: FormComponentId, detail: AddressDetail) extends Expr
 final case class PeriodFun(dateCtx1: Expr, dateCtx2: Expr) extends Expr
 
+sealed trait FunctionProp
```

Comment:
`FunctionProp `sounds more generic than it actually is. Why not to call it `PeriodFunctionProp`? Or perhaps just `PeriodFn`?